_layouts/default.html

```diff
@@ -5,10 +5,15 @@
 <body class="home-template">
     <div class="site-wrapper">
         {% include header.html %}
-        {{ content }}        
+        {{ content }}
         {% include footer.html %}
     </div>
 </body>
 
 <script src="{{ site.github.url | prepend: site.baseurl}}/js/global.js"></script>
+<script type="text/javascript">
```

Comment:
Please place this script into `post_scripts.html` because the js logic for comments is only implemented in `post.js`